design.md

```diff
@@ -1,13 +1,37 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+The target for the design is users who want to program Picobot and have either a basic knowledge of 
+programming or logic flow.
 
```

Comment:
That's what we also went for. It's interesting but definitely unsurprising that we had a similar end user in mind and both of our designs had a very Python-y feel to them.